src/webrtc/groupCall.ts

```diff
@@ -509,6 +518,31 @@ export class GroupCall extends TypedEventEmitter<
         return true;
     }
 
+    public async setVadMicrophoneMuted(muted: boolean): Promise<boolean> {
+        // hasAudioDevice can block indefinitely if the window has lost focus,
+        // and it doesn't make much sense to keep a device from being muted, so
+        // we always allow muted = true changes to go through
+        if (!(await this.client.getMediaHandler().hasAudioDevice())) {
+            return false;
+        }
```

Comment:
We need to check `muted` here, like in the other methods

Reply:
I have a check on Callfeed, or rather muting manually turns vad off, as we do not need to check whether or not the user is loud enough when we are muted either way. 
This means this function won't be called when you are muted manually.